MathParser/LocalizedNumberExtractor.swift

```diff
@@ -58,9 +58,17 @@ internal struct LocalizedNumberExtractor: TokenExtractor {
         return .value(token)
     }
     
+    /// - Returns: True if the string is a valid prefix of a localized number.
+    private func isValidPrefix(_ string: String) -> Bool {
+        if string == decimalNumberFormatter.decimalSeparator {
+            return true
+        }
+        
+        return false
```

Comment:
Let's change this to `return string == decimalNumberFormatter.decimalSeparator`

Reply:
Done.